prebuilt_forms/species_details_2.php

```diff
@@ -156,6 +160,64 @@ public static function get_parameters() {
           'default' => 'one_page',
           'group' => 'User Interface',
         ],
+        // Taxon selector
```

Comment:
full stop at end of comment (Drupal standards are fussy!)

Reply:
I am linting to Drupal standards - but for some reason that isn't flagged.

Reply:
Odd - I wonder if the standards have changed since I installed. How is your PHPCS installed and what versions of that and the standards are you using?

Reply:
I'm working in a linux vm with VS Code. I installed the VS phpcs plugin (Shevaua v1.0.8) and installed phpcs globally using composer (composer require squizlabs.php_codesniffer) and also installed the Drupal coding standards globally with composer (composer global require drupal/coder).

Reply:
Here's the output of `composer global show`
```
dealerdirect/phpcodesniffer-composer-installer v0.7.2  PHP_CodeSniffer Standards Composer Installer Plugin
drupal/coder                                   8.3.16  Coder is a library to review Drupal code.
phpstan/phpdoc-parser                          1.13.1  PHPDoc parser with support for nullable, intersection and generic types
sirbrillig/phpcs-variable-analysis             v2.11.9 A PHPCS sniff to detect problems with variables.
slevomat/coding-standard                       8.6.4   Slevomat Coding Standard for PHP_CodeSniffer complements Consistence Coding Standard by providing sni...
squizlabs/php_codesniffer                      3.7.1   PHP_CodeSniffer tokenizes PHP, JavaScript and CSS files and detects violations of a defined set of co...
symfony/deprecation-contracts                  v2.5.2  A generic function and convention to trigger deprecation notices
symfony/polyfill-ctype                         v1.27.0 Symfony polyfill for ctype functions
symfony/yaml                                   v5.4.16 Loads and dumps YAML files
```

Reply:
Strange. I'm using global composer as well and I was on minor versions slightly before yours but not significantly different, so I've done a composer global up and now have the same versions as you, but still get the warning about not finishing comments with a full stop.

Reply:
```
installed_paths: ../../drupal/coder/coder_sniffer,../../sirbrillig/phpcs-variable-analysis,../../slevomat/coding-standard
```

Reply:
Hm - those are the same three folders that mine is set to. I'll just have to pay extra care to comments for now.

Reply:
That's a shame - thanks for trying. The 3 rules I get picked up on are:
1. Max width 80 characters
2. Full-stop at end.
3. Start with uppercase.
4. Actually there's a 4th but I don't often get caught on this one - space after the // .

Reply:
Thanks John. It's strange - the max size rule *is* applied but none of the others are.